libshpool/src/daemon/pager.rs

```diff
@@ -128,6 +130,7 @@ impl Pager {
         msg_file.write_all(cleaned_msg.as_slice()).context("writing msg to tmp pager file")?;
 
         let mut cmd = process::Command::new(&self.pager_bin);
+        cmd.env_clear().envs(shell_env.to_vec());
```

Comment:
(no action needed)
just making sure i understand, unlike the shell where we inject the env, here we clear and insert only the shpool-created env variables. and this is to satistfy the TERM override issue mentioned in the description

i like this separation and how easy it is to read what is done

Reply:
Yeah pretty much. I think it might not be horrible to inherit from the daemon, but this way seems a little more predictable.